comp/core/status/statusimpl/status.go

```diff
@@ -167,7 +166,7 @@ func (s *statusImplementation) GetStatus(format string, verbose bool, excludeSec
 
 		return json.Marshal(stats)
 	case "text":
-		var b = new(bytes.Buffer)
+		b := new(bytes.Buffer)
```

Comment:
👏

Reply:
Unfortunately I can't take credit for this, it was the linter.

Reply:
I would always think it was you 😉